support-workers/src/main/scala/com/gu/emailservices/ContributionEmailFields.scala

```diff
@@ -53,6 +53,7 @@ class ContributionEmailFields(
       ))
       case _: PayPalReferenceTransaction => Future.successful(List("payment method" -> "PayPal"))
       case _: CreditCardReferenceTransaction => Future.successful(List("payment method" -> "credit / debit card"))
+      case _: SepaPaymentMethod => Future.successful(List("payment method" -> "Sepa"))  // TODO
```

Comment:
any more detail on that TODO?  does that just mean the thank you email in braze needs doing?

Reply:
Oops, missed this one! One of the remaining tasks for SEPA is to make the emails work (the email is currently blank), so we'll address this soon